src/home/RegisteredView.tsx

```diff
@@ -129,14 +121,16 @@ export function RegisteredView({ client, isPasswordlessUser }: Props) {
       <div className={commonStyles.container}>
         <main className={commonStyles.main}>
           <HeaderLogo className={commonStyles.logo} />
-          <CallTypeDropdown callType={callType} setCallType={setCallType} />
+          <Heading size="lg" weight="semibold">
+            {t("Start new call")}
+          </Heading>
           <Form className={styles.form} onSubmit={onSubmit}>
             <FieldRow className={styles.fieldRow}>
               <InputField
                 id="callName"
                 name="callName"
-                label={callNameLabel}
-                placeholder={callNameLabel}
+                label={t("Name of call")}
+                placeholder={t("Name of call")}
```

Comment:
I chose these strings, because they're what's used in the new designs

Reply:
Also useful because this really is for starting calls now, not joining them.